ghproxy/ghmetrics/ghmetrics.go

```diff
@@ -93,6 +93,9 @@ func init() {
 // `github_token_usage` as well as `github_token_reset` on prometheus.
 func CollectGitHubTokenMetrics(tokenHash, apiVersion string, headers http.Header, reqStartTime, responseTime time.Time) {
 	remaining := headers.Get("X-RateLimit-Remaining")
+	if remaining == "" {
+		return
```

Comment:
Should we log a debug message so that we can generate some data on how often this happens?

Reply:
We can look at the data from the last N weeks this has been live and/or all the zero-blips ... unclear if it's useful.